src/main/java/com/uid2/shared/auth/OperatorKey.java

```diff
@@ -34,6 +35,18 @@ public OperatorKey(String key, String name, String contact, String protocol, lon
         this.created = created;
         this.disabled = disabled;
         this.siteId = siteId;
+        this.roles = new HashSet<>(Arrays.asList(Role.OPERATOR));
+    }
+    public OperatorKey(String key, String name, String contact, String protocol, long created, boolean disabled, Integer siteId, Set<Role> roles) {
+        this.key = key;
+        this.name = name;
+        this.contact = contact;
+        this.protocol = protocol;
+        this.created = created;
+        this.disabled = disabled;
+        this.siteId = siteId;
+        roles.add(Role.OPERATOR);
+        this.roles = Collections.unmodifiableSet(roles);
```

Comment:
Since we should be using Java 11, `Set.of()` might be a better choice than `Collections.unmodifiableSet()` since the former doesn't allow null values.

Reply:
@gmsdelmundo-ttd I think Set.of() only takes a collection of elements but the roles here is a Set, could we use Set.of?